### PR TITLE
Fixed monsters outfits

### PR DESCRIPTION
**Pull request by ghost**
- fixed a lot of outfit colors
- added a few missing monsters schemas
- added wrapable key to golden outfit display
- fixed unused purple elf body
- added bag with stolen gold and the bosses that drop it
- removed duplicated entries on monsters.xml file
- removed a unused file (efreet) and added a few entries that were present as xml but not registered on monsters.xml
- moved a few files to their respectives correct places
- removed a custom unused monster
- fixed lots of corpses ID from 12.00 update

### CODE REVIEW

File: data/monster/Elves/crazed winter vanguard.xml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+		<element earthPercent="-15"/>
+		<element icePercent="50"/>
+		<element deathPercent="15"/>
+	</elements>
+	<immunities>
+		<immunity invisible="1"/>
+	</immunities>
+	<loot>
+		<item name="platinum coin" countmax="13" chance="100000"/>
+		<item name="red crystal fragment" chance="1000000"/>
```
Comment (vankk) [inline data/monster/Elves/crazed winter vanguard.xml]: Too high chance, should be 100k max.

File: data/monster/Elves/crazed winter vanguard.xml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+		<item name="red gem" chance="1000000"/>
```
Comment (vankk) [inline data/monster/Elves/crazed winter vanguard.xml]: Too high chance, should be 100k max.

File: data/monster/Elves/crazed summer rearguard.xml [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+	<immunities>
+		<immunity invisible="1"/>
+		<immunity paralyze="1"/>
+	</immunities>
+	<voices interval="10000" chance="7">
+		<voice sentence="Is this real life?"/>
+		<voice sentence="Weeeuuu weeeuuu!!!"/>
+	</voices>
+	<loot>
+		<item name="power bolt" chance="1000000"/>
```
Comment (vankk) [inline data/monster/Elves/crazed summer rearguard.xml]: High chance too.

File: data/monster/Elves/crazed summer rearguard.xml [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+		<item name="small enchanted sapphire" countmax="2" chance="1000000"/>
```
Comment (vankk) [inline data/monster/Elves/crazed summer rearguard.xml]: High chance too.

File: data/monster/Elves/crazed summer rearguard.xml [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+		<item name="platinum coin" countmax="11" chance="1000000"/>
```
Comment (vankk) [inline data/monster/Elves/crazed summer rearguard.xml]: High chance too.

File: data/monster/Elves/crazed summer vanguard.xml [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+	<immunities>
+		<immunity invisible="1"/>
+		<immunity paralyze="1"/>
+	</immunities>
+	<voices interval="10000" chance="7">
+		<voice sentence="I see colours, all colours! Or are these just illusions?"/>
+		<voice sentence="La di da di doo!!"/>
+	</voices>
+	<loot>
+		<item name="platinum coin" countmax="13" chance="1000000"/>
```
Comment (vankk) [inline data/monster/Elves/crazed summer vanguard.xml]: High chance too.

File: data/monster/Elves/crazed summer vanguard.xml [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+		<item name="elven astral observer" countmax="2" chance="1000000"/>
```
Comment (vankk) [inline data/monster/Elves/crazed summer vanguard.xml]: High chance too.

File: data/monster/Elves/crazed winter rearguard.xml [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+	</defenses>
+	<elements>
+		<element firePercent="20"/>
+	</elements>
+	<immunities>
+		<immunity invisible="1"/>
+	</immunities>
+	<loot>
+		<item name="platinum coin" countmax="5" chance="100000"/>
+		<item name="red crystal fragment" chance="1000000"/>
```
Comment (vankk) [inline data/monster/Elves/crazed winter rearguard.xml]: High chance too.

File: data/monster/Elves/crazed winter rearguard.xml [hunk 9]
```diff
@@ -0,0 +1,62 @@
...
+		<item name="red gem" chance="1000000"/>
```
Comment (vankk) [inline data/monster/Elves/crazed winter rearguard.xml]: High chance too.

**Review by vankk — COMMENTED** [review]
Still there is some items that was a high chance yet.

**Comment by EPuncker** [comment]
por favor guys agilizem os merges, sei q vcs tem outras coisas pra fazer e tals, mas com o tanto de PR q ta sendo feita fica complicado, pois vai gerar muito conflito se forem deixadas tantas abertas assim, a minha é só mudança simples, já testei no console e tem zero erros, to só vendo todas PR depois da minha quebrando a minha :(

**Pull request closed without merging**